docs/conf.py

```diff
@@ -116,10 +115,10 @@ def get_release():
 
 # -- Options for HTML output ----------------------------------------------
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
+# The theme to use for HTML and HTML Help pages. See the documentation for
 # a list of builtin themes.
 if IS_READTHEDOCS:
-    # equals to default RTD theme
+    # equals to the default RTD theme
```

Comment:
I think we can set html_theme to `sphinx_rtd_theme` regardless of environment. If it is built on RTD, it will then be built with the exact version locked in our python dependencies too. The local result should be closer to production.